manim/scene/scene.py

```diff
@@ -781,8 +781,8 @@ def play_internal(self, *args, **kwargs):
 
         Parameters
         ----------
-        *args : Animation or mobject with mobject method and params
-        **kwargs : named parameters affecting what was passed in *args e.g run_time, lag_ratio etc.
+        \*args : Animation or mobject with mobject method and params
+        \*\*kwargs : named parameters affecting what was passed in \*args e.g run_time, lag_ratio etc.
```

Comment:
The colon notation would only be used to specify a type of the argument. That's not really possible here (actually, the form this is documented is far from ideal), but it is as it is. :-)

```suggestion
        args
            Animation or mobject with mobject method and params
        kwargs
            named parameters affecting what was passed in ``args`` e.g. ``run_time``, ``lag_ratio`` etc.
```